sig-scalability/slos/pod_startup_latency.md

```diff
@@ -4,29 +4,43 @@
 
 | Status | SLI | SLO |
 | --- | --- | --- |
-| __Official__ | Startup latency of stateless<sup>[1](#footnote1)</sup> and schedulable<sup>[2](#footnote2)</sup> pods, excluding time to pull images and run init containers, measured from pod creation timestamp to when all its containers are reported as started and observed via watch, measured as 99th percentile over last 5 minutes | In default Kubernetes installation, 99th percentile per cluster-day <= 5s |
+| __Official__ | Startup latency of schedulable<sup>[1](#footnote1)</sup> stateless<sup>[2](#footnote2)</sup> pods, excluding time to pull images and run init containers, measured from pod creation timestamp to when all its containers are reported as started and observed via watch, measured as 99th percentile over last 5 minutes | In default Kubernetes installation, 99th percentile per cluster-day <= 5s |
+| __WIP__ | Startup latency of schedulable<sup>[1](#footnote1)</sup> stateful<sup>[3](#footnote3)</sup> pods, excluding time to pull images, run init containers and provision volumes (in delayed binding mode), measured from pod creation timestamp to when all its containers are reported as started and observed via watch, measured as 99th percentile over last 5 minutes | In default Kubernetes installation, 99th percentile per cluster-day <= X where X depends on storage provider |
```

Comment:
How do you intend to exclude volume provisioning time when it is pod creation that triggers volume provisioning ("delayed binding mode")?

Reply:
1. In tests it's easy - we simply won't use the "delayed binding mode" :-)
When monitoring production clusters, it's TBD.

Reply:
Do we plan to run the tests in regional clusters?  If so, then if we manually provision the volumes before the test, we need to make sure they are balanced across all zones so that we don't make pods unschedulable due to insufficient cpu/memory resources in a zone.

Reply:
Added a note to test scenario section.